frontend/appflowy_flutter/packages/flowy_infra/lib/theme.dart

```diff
@@ -21,6 +32,7 @@ class AppTheme {
   });
 
   factory AppTheme.fromName(String themeName) {
+    if(!BuiltInTheme.themes.contains(themeName)) return fallbackTheme;
```

Comment:
```suggestion
    if (!BuiltInTheme.themes.contains(themeName)) {
      return fallbackTheme;
    }
```
IIRC, effective dart recommends that all `if` statements have a block.